### ISSUE TITLE
Text to .json chunks

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#17]
## User story
1. As a developer
2. I need to break down the text file into .json chunks
3. So that the data is ready to be ingested by the LLM.

## Acceptance criteria
* The text files are broken apart into .json chunks of a size that gets accepted by the LLM.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
